custom/icds_reports/data_pull/sql_queries/vhsnd_monthly_report.sql

```diff
@@ -0,0 +1,15 @@
+SELECT state_name,
+    district_name,
+    block_name,
+    supervisor_name,
+    awc_name,
+    awc_site_code,
+    vhsnd_date_past_month
+FROM agg_awc_monthly INNER JOIN "ucr_icds-cas_static-vhnd_form_28e7fd58" vhnd_ucr ON agg_awc_monthly.awc_id=vhnd_ucr.awc_id AND vhnd_ucr.vhsnd_date_past_month>='{from_date}' AND vhnd_ucr.vhsnd_date_past_month<'{till_date}'
+WHERE agg_awc_monthly.aggregation_level=5 AND agg_awc_monthly.month='{from_date}' AND agg_awc_monthly.state_is_test<>1 AND agg_awc_monthly.num_launched_awcs=1 ORDER BY state_name,
+    district_name,
+    block_name,
+    supervisor_name,
+    awc_name,
```

Comment:
nit: i think you can avoid `_name` to these 5 columns.

Reply:
these ar the columns from agg_awc_monthly.i believe we need them

Reply:
sorry, i meant if you would do state_name as state in select itself, then you can change these as well. Fine to ignore this, just thought it was explicit that these were names only.